config/config.go

```diff
@@ -71,6 +71,7 @@ func init() {
 	cfg.SetDefault("agent.topology.netlink.metrics_update", 30)
 	cfg.SetDefault("agent.topology.neutron.domain_name", "Default")
 	cfg.SetDefault("agent.topology.neutron.endpoint_type", "public")
+	cfg.SetDefault("agent.topology.neutron.ssl_skip_verify", false)
```

Comment:
ssl_insecure would fit better, like curl and skydive tls options

Reply:
Changed